qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -243,7 +243,7 @@ def run(self, dag: DAGCircuit) -> DAGCircuit:
             Output dag with UnitaryGates synthesized to target basis.
 
         Raises:
-            TranspilerError: if a 'method' was specified for the class and is not
+            TranspilerError: if ``'method'`` was specified for the class and is not
```

Comment:
This one was either already fine, or the inner quotes in the changed version should be dropped.

Reply:
inner quotes removed in https://github.com/Qiskit/qiskit-terra/pull/9219/files/70bcae5d2d812014502c7b1aea6f6f4eee26e45a#r1038036824 thanks!